db/views/hmis_households_v01.sql

```diff
@@ -0,0 +1,27 @@
+-- To update, run rails generate scenic:view hmis_households --replace
+
+SELECT
+	"Enrollment"."HouseholdID" as id,
+  "Enrollment"."HouseholdID" as "HouseholdID",
+	MAX("Enrollment"."ProjectID") as "ProjectID",
+  MAX("Enrollment"."data_source_id") as data_source_id,
+	MIN("Enrollment"."EntryDate") as earliest_open,
+	CASE
+		WHEN BOOL_OR("Exit"."ExitDate" is NULL) THEN NULL
+		ELSE MAX("Exit"."ExitDate")
+	END as latest_exit,
+	BOOL_OR("Enrollment"."ProjectID" is NULL) as any_wip,
+  CASE
+		WHEN BOOL_OR("Enrollment"."DateDeleted" is NULL) THEN NULL
+		ELSE MAX("Enrollment"."DateDeleted")
+	END as "DateDeleted",
```

Comment:
Hmm, I don't think we want to include deleted enrollments/households in the view.

Can you add a `AND "Enrollment"."DateDeleted" IS NULL` to the where clause, and just put `NULL AS "DateDeleted",` here?